jetbrains_projects/__init__.py

```diff
@@ -71,15 +71,11 @@ def _parse_recent_projects(self, recent_projects_file: Path) -> list[Project]:
                 project_path = entry.attrib["key"].replace("$USER_HOME$", str(Path.home()))
 
                 tag_opened = entry.find(".//option[@name='projectOpenTimestamp']")
-                last_opened = int(
-                    tag_opened.attrib["value"]
-                    if tag_opened is not None and "value" in tag_opened.attrib
-                    else None
-                )
+                last_opened = tag_opened.attrib["value"] if tag_opened is not None and "value" in tag_opened.attrib else None
 
                 if project_path and last_opened:
                     projects.append(
-                        Project(name=Path(project_path).name, path=project_path, last_opened=last_opened)
+                        Project(name=Path(project_path).name, path=project_path, last_opened=int(last_opened))
```

Comment:
I would say that:
- it's working locally, so seems fine :smile: ("works on my machine")
- and `last_opened` will be `None` given the tag's value was not found, so the if clause protects against it.

Reply:
The if clause 😄 sure. I still don't have a proper strategy when to increase versions. But I guess since I will release this at least the minor version should increase. What do you think. Major Version increase then if something substantial and user noticeable happens.

Reply:
Hi @ManuelSchneid3r . I incremented the minor (from 1.1 to 1.2).
I was thinking the version number would be semantic, which is not terribly useful for a plugin, but at least "don't make me think". Simpler like the current one if fine, and somehow clear : increment minor on feat/fix, major on overhaul change or breaking.